include/actor_types.h

```diff
@@ -5,6 +5,120 @@
 // Made this because there will likely be a lot of structs in this file that will be compiled into one eventually.
 // This keeps the other files cleaner.
 
+#define ACTOR_FALLING_ROCK        0x05
+#define ACTOR_BANANA              0x06
+#define ACTOR_GREEN_SHELL         0x07
+#define ACTOR_RED_SHELL           0x08
+#define ACTOR_YV_EGG              0x09 // YV -> Yoshi Valley
+#define ACTOR_PIRNHA_PLANT        0x0A
+#define ACTOR_ITEM_BOX            0x0C
+#define ACTOR_FAKE_ITEM_BOX       0x0D
+#define ACTOR_BANANA_BUNCH        0x0E
+#define ACTOR_TRAIN_ENGINE        0x0F
+#define ACTOR_TRAIN_COAL_CAR      0x10
+#define ACTOR_TRAIN_PASSENGER_CAR 0x11
+#define ACTOR_TRIPLE_GREEN_SHELL  0x15
+#define ACTOR_TRIPLE_RED_SHELL    0x16
+#define ACTOR_MRW_SIGN            0x17 // MRW -> Mario Raceway
+#define ACTOR_WS_SIGN             0x23 // WS -> Wario Stadium
+#define ACTOR_PADDLE_WHEEL        0x26 // Just the paddle wheel, or does it mean the entire boat?
+#define ACTOR_RR_CROSSING         0x27 // RR -> Railroad
+#define ACTOR_BLUE_SHELL          0x2A
+#define ACTOR_HAB_ITEM_BOX        0x2B // HAB -> Hot Air Baloon
+#define ACTOR_KIWANO_FRUIT        0x2D
+
+/*
+Used by the engine, coal car, and passenger car
+*/
+struct train_car {
+    /* 0x00 */ s16 type;
+    /* 0x02 */ s16 flags;
+    /* 0x04 */ s16 unk_04;
+    /* 0x06 */ s16 wheelRot;
+    /* 0x08 */ f32 unk_08;
+    /* 0x0C */ f32 unk_0C;
+    /* 0x10 */ Vec3s rot;
+    /* 0x10 */ s16 unk_16;
+    /* 0x18 */ Vec3f pos;
+    /* 0x24 */ Vec3f velocity;
+    /* 0x30 */ UnkActorInner unk30;
+}; // size = 0x70
+
+struct railroad_crossing {
+    /* 0x00 */ s16 type;
+    /* 0x02 */ s16 flags;
+    /* 0x04 */ s16 someTimer;
+    /* 0x06 */ s16 crossingId;
+    /* 0x08 */ f32 unk_08;
+    /* 0x0C */ f32 unk_0C;
+    /* 0x10 */ Vec4s unk_10;
+    /* 0x18 */ Vec3f pos;
+    /* 0x24 */ Vec3f velocity;
+    /* 0x30 */ UnkActorInner unk30;
+}; // size = 0x70
+
+struct falling_rock {
+    /* 0x00 */ s16 type;
+    /* 0x02 */ s16 flags;
+    /* 0x04 */ s16 respawnTimer;
+    /* 0x06 */ s16 unk_06;
+    /* 0x08 */ f32 unk_08;
+    /* 0x0C */ f32 boundingBoxSize;
+    /* 0x10 */ Vec3s rot;
+    /* 0x16 */ s16 unk_16;
+    /* 0x18 */ Vec3f pos;
+    /* 0x24 */ Vec3f velocity;
+    /* 0x30 */ UnkActorInner unk30;
+}; // size = 0x70
+
+struct yv_egg {
```

Comment:
Can we please do long names for everything. Other readers will see "yv_egg" out of context and not understand it.